lib/ButtonNew/ButtonAvatar/buttonAvatar.scss

```diff
@@ -0,0 +1,31 @@
+.button-avatar {
+  @apply rounded-full p-0 border-transparent relative;
+  width: 34px;
+  height: 34px;
```

Comment:
💚 If we can make these work with REM values, the avatar will scale better if the user zooms in and out.

Reply:
Ah, maybe - just realized this is styling the button, not the avatar!